wsktools/WhiskSwiftTools/ProjectReader.swift

```diff
@@ -498,8 +498,9 @@ open class ProjectReader {
                     if let item = actionsDict[(prefix+itemName) as NSString] {
                         var runtime = item.runtime
                         if let kind = action["kind"] as? String {
-                            if runtime == Runtime.swift3 && kind == "swift:3" {
-                                runtime = Runtime.swift3
+                            // swift:default == swift:3 at the moment
+                            if runtime == Runtime.swift && kind == "swift:3" {
+                                runtime = Runtime.swift
```

Comment:
Holdover (and future proofing?) for when we have more than one Swift runtime.  This line allows you to override the default value using the manifest file.

A lot of this is now part of `wskdeploy` -- if at some point we plan to move this tool forward I would strip away all this and focus on only xcode specific things.